app/docs/10-localization.md

```diff
@@ -0,0 +1,21 @@
+# Localization
+
+For localization needs Report Portal uses [react-intl](https://github.com/yahoo/react-intl) library, which is a part of [FormatJS](https://formatjs.io/).
+
+[Documentation](https://github.com/yahoo/react-intl/wiki)
+
+All text content of application should be controlled by localization system.
+At the moment Report Portal supports English (default), Russian and Belorussian languages.
+
+Localization message id format: `ComponentsName.elementName` (for example FiltersPage.addFilterButton)
+
+Localization of currently realised (Backbone) version you can find [here](https://github.com/reportportal/service-ui/tree/localization_by/src/main/resources/public/js/src/localizations).
```

Comment:
1. realised -> implemented
2. As far as I understand this branch is out of date. Maybe it's good to mention that this branch is for BY localization only and give another link to the fresh localization folder from `develop` branch?

Reply:
What do you think about adding another note telling that this branch is not consistent with the current `develop` branch and should be used for getting BY localization? For other langs it's better to look in the `develop` branch.

Reply:
ok